mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
-
+function introduceMe(name, age) {
+  return `Hello, my name is ${name} and I am ${age} years old`;
+}
 function getTotal(a, b) {
-  total = a ++ b;
+  total = a + b;
```

Comment:
This is missing a declaration, could you explain why it worked without a `const` or `let` declaration and why it would be a bad practice, even though it works?

Reply:
i made it global variable by mistake, but has a side effect it is it may cause unexpected behaviour